### DISCUSSION TITLE
Overwrite the To/From fields when bulk change

### DISCUSSION THREAD

**Discussion by vozmor** [TumblThreeApp/TumblThree#499] [General]
When you select multiple blogs (rows), you could be write in the From and To fields even if one ore more of the selected blogs already have a date in either or both.

If you only select blogs whose To/From fileds are empty, you can bulk change them.

Now, if there is just one single blog among the selected blogs, a blog that has a To or From field that already includes a value, the To/From box greys out, so you cannot bulk change the selected blogs.

I would like to be able to bulk change multiple blogs, that is, the To/From fields should be remain writeable, and these fields should be overwritten by a bulk change.

Thanks.

Vozzy

**Comment by thomas694** [comment]
By design the user shall not accidentally overwrite a (different) value which he's not aware of. For the checkboxes, there's this special state symbolizing that some are checked others not, but such a thing doesn't exist for textboxes by default.
We would need to use a colored border, a different background color, a warning message or something like that.

For now I move it to general discussion. Suggestions are welcome.
